FirebaseCore/Sources/Private/FIRHeartbeatLogger.h

```diff
@@ -0,0 +1,36 @@
+// Copyright 2021 Google
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <Foundation/Foundation.h>
+
+#if SWIFT_PACKAGE
+@import HeartbeatLogging;
+#else
```

Comment:
currently failing on CI
```
.../FirebaseCore/Sources/Private/FIRHeartbeatLogger.h:18:1: use of '@import' when C++ modules are disabled, consider using -fmodules and -fcxx-modules
```

Only seems to surface when building Firebase Package

Reply:
Because Firestore's C++ sources are importing the private headers.